docs/_data/components.yml

```diff
@@ -24,6 +24,9 @@
     - title: Image
       meta: image
 
+    - title: Info Label
```

Comment:
Not sure if we should do that, since we probably have no tile image for `Info Label` @droshev ?